scripts/productaanvraag/notify.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+#
+# SPDX-FileCopyrightText: 2025 INFO.nl
+# SPDX-License-Identifier: EUPL-1.2+
+#
+set -e
+
+help()
+{
+   echo "Notifies ZAC about a product request. Note that the ZAC used endpoint requires API key authentication."
```

Comment:
Nice! But should the script not also create the productaanvraag object in Objecten in the first place? Instead of provisioning it through the demodata.json? That way you are more free to create any productaanvraag you wish. Still, good start this. :-)

Reply:
Ok. Have working first version